operator/pkg/model/ingestion/gateway.go

```diff
@@ -370,6 +372,23 @@ func toTimeout(timeouts *gatewayv1.HTTPRouteTimeouts) model.Timeout {
 	return res
 }
 
+func toHTTPRetry(retry *gatewayv1.HTTPRouteRetry) *model.HTTPRetry {
```

Comment:
please move the HTTP Retry changes into the actual commit. otherwise this is spread over both commits.

Reply:
good catch :+1: